frontend/app/routes/_public+/apply+/$id+/dental-insurance.tsx

```diff
@@ -54,94 +56,93 @@ export async function action({ request, params }: ActionFunctionArgs) {
-        </li>
-        <li>
-          <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.list.dental-coverage" />
-        </li>
+        <li>{t('dental-insurance.list.employment')}</li>
+        <li>{t('dental-insurance.list.pension')}</li>
+        <li>{t('dental-insurance.list.purchased')}</li>
+        <li>{t('dental-insurance.list.professional')}</li>
         <li className="list-none">
-          <Details id={t('apply:dental-insurance-question.detail.additional-info.title')} title={t('apply:dental-insurance-question.detail.additional-info.title')}>
+          <CollapsibleDetails id={t('dental-insurance.detail.additional-info.title')} summary={t('dental-insurance.detail.additional-info.title')}>
             <div>
-              <p>{t('apply:dental-insurance-question.detail.additional-info.eligible')}</p>
+              <Trans ns={handle.i18nNamespaces} i18nKey="dental-insurance.detail.additional-info.not-eligible" />
+              <p className="mt-4">{t('dental-insurance.detail.additional-info.eligible')}</p>
               <ul className="mb-4 list-disc pl-6">
-                <li>{t('apply:dental-insurance-question.detail.additional-info.list.opted')}</li>
-                <li>{t('apply:dental-insurance-question.detail.additional-info.list.cannot-opt')}</li>
+                <li>{t('dental-insurance.detail.additional-info.list.opted')}</li>
+                <li>{t('dental-insurance.detail.additional-info.list.cannot-opt')}</li>
               </ul>
-              <Trans ns={handle.i18nNamespaces} i18nKey="apply:dental-insurance-question.detail.additional-info.not-eligible" />
             </div>
-          </Details>
+          </CollapsibleDetails>
         </li>
       </ul>
     </>
   );
 
   return (
-    <Form method="post">
-      {options.length > 0 && (
-        <div className="my-6">
-          {options.length > 0 && (
-            <InputRadios
-              id="dental-insurance"
-              name="dentalInsurance"
-              legend={t('apply:dental-insurance-question.legend')}
-              options={options.map((option) => ({
-                children: (
-                  <div>
-                    <strong>{getNameByLanguage(i18n.language, option)}</strong>
-                    {`, ${option.id === 'yes' ? t('apply:dental-insurance-question.option-yes') : t('apply:dental-insurance-question.option-no')}`}
-                  </div>
-                ),
-                value: option.id,
-                defaultChecked: state.dentalInsurance?.dentalInsurance === option.id,
-              }))}
-              helpMessagePrimary={helpMessage}
-              helpMessagePrimaryClassName="pl-8 text-black"
-            />
-          )}
+    <>
+      {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+      <Form method="post">
+        {options.length > 0 && (
+          <div className="my-6">
+            {options.length > 0 && (
```

Comment:
No need to check `options.length > 0` again here since it was checked two lines up.